src/Core/Domain/OrderReturn/Command/BulkDeleteProductFromOrderReturnCommand.php

```diff
@@ -34,7 +34,7 @@
 use PrestaShop\PrestaShop\Core\Domain\OrderReturn\ValueObject\OrderReturnId;
 
 /**
- * Deletes products from given order return.
+ * Deletes products from given order return. Order return is called merchandise return in user interface.
```

Comment:
I doubt that it should be mentioned where is the command called from :thinking: In theory It could be called from anywhere